test/test_ops.py

```diff
@@ -250,6 +249,7 @@ def __new__(cls, name, variant_test_name=""):
 
             # Failing Ops
             # Refer for more info : https://github.com/pytorch/xla/pull/3019#issuecomment-877132385
+            # AllowedOpInfoEntry('einsum'), https://github.com/pytorch/xla/issues/4052
```

Comment:
@steventk-g, wasn't the intention of falling back to cpu so that we don't have to disable the `einsum` tests?

Reply:
We still need to remove the test because precision tests for float32 are also failing

Reply:
Then can we also fall back on those types? If this test was passing before the new `einsum` lowering, we should still expect this test to pass with the lowering implementation.